src/components/modules/AverageTimeReports/AverageTimeReports.tsx

```diff
@@ -0,0 +1,104 @@
+/*eslint-disable*/
+import moment from 'moment';
+import React, { useEffect, useState } from 'react';
+import { useForm } from 'react-hook-form';
+import styled from 'styled-components';
+
+import { type AirQuality } from '@/components/modules/AverageTimeReports/types';
+import data from '@/mocks/avg-data.json';
+
+type FormValues = {
+  dateFrom: string;
+  dateTo: string;
+};
+
+const DATE_FORMAT = 'YYYY-MM-DD';
+
+const StyledForm = styled.form`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 20px;
+`;
+
+const StyledInput = styled.input`
+  margin-bottom: 10px;
+`;
+
+const StyledButton = styled.button`
+  margin-top: 20px;
+  margin-bottom: 20px;
+`;
+
+const StyledList = styled.ul`
+  display: flex;
+  flex-wrap: wrap;
+  list-style: none;
+  padding: 0;
+  margin-top: 20px;
+  width: 50%;
+  margin: auto;
+`;
+
+const StyledListItem = styled.li`
+  flex-basis: 50%;
+  padding: 5px;
+  border: 1px solid #000;
+  text-align: center;
+`;
+
+const StyledHeader = styled.h1`
+  text-align: center;
+`;
+
+function AverageTimeReports() {
+  const { register, handleSubmit, formState } = useForm<FormValues>({
+    defaultValues: {
+      dateTo: moment().format(DATE_FORMAT),
+      dateFrom: moment().format(DATE_FORMAT)
+    }
+  });
+
+  const [avgData, setAvgData] = useState<AirQuality | null>(null);
+  const [submitted, setSubmitted] = useState<boolean>(false);
+
+  const { errors } = formState;
+
+  useEffect(() => {
+    setAvgData(data);
+  }, []);
+
+  const onSubmit = (formData: FormValues): void => {
+    // @ts-expect-error
+    const { dateFrom, dateTo } = formData;
+    // @TODO fetch from API and set to chartData
```

Comment:
```suggestion
    // TODO fetch from API and set to chartData
```
I think that you should leave TODO without `@` before it. VS Code extensions displaying TODOs search for the `TODO` keyword by default and will not find `@TODO`.